apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.sql

```diff
@@ -0,0 +1,23 @@
+-- BLOCK select_draft_generation_info_by_course_id
+SELECT
+  dqm.*,
+  q.qid,
+  u.uid
+FROM
+  questions AS q
+  LEFT JOIN draft_question_metadata AS dqm ON dqm.question_id = q.id
+  LEFT JOIN users As u ON u.user_id = dqm.created_by
+WHERE
+  q.course_id = $course_id
```

Comment:
any draft question will show (even if it is manually inserted); other columns will only show if they're recognized. we can change display format if needed.